### DISCUSSION TITLE
Enable image resizing in Expendables

### DISCUSSION THREAD

**Discussion by Loriel-S-Parloa** [GitbookIO/community#270] [Feature Requests]
I appreciate the addition of image resizing functionality. It would be beneficial if this feature could also be extended to images pasted within Expendables. Currently, these images tend to be too small and there is a lack of options to adjust their size.
Thank you!

**Comment by aikoii** [comment]
Thanks for the suggestion 🙂! I definitely see how this would be helpful. Linking to #16
